src/vm/jitinterface.h

```diff
@@ -22,6 +22,14 @@
 #define MAX_UNCHECKED_OFFSET_FOR_NULL_OBJECT ((GetOsPageSize() / 2) - 1)
 #endif // !FEATURE_PAL
 
+
+enum StompWriteBarrierXXXCompletion
```

Comment:
Well, I haven't found any guides regarding naming such stuff, also I'm not sure this is the right place for this decl.

Reply:
As good a place as any, I think. My only feedback would be to prefix the enum members with something related to the enum name, e.g. `SWB_PASS`, just to avoid pollulting the global namespace with bare identifiers. Jan might have more an opinion on naming.